_dashboards/visualize/vega.md

```diff
@@ -190,3 +194,122 @@ The following is an example Vega specification with `Demo US Cluster` as the spe
 }
 ```
 {% include copy-curl.html %}
+
+## Creating Vega visualizations with OpenSearch Dashboards
+
+Before proceeding, ensure that the following configuration is enabled in your `opensearch_dashboards.yml` file:
+
+```yml
+vis_type_vega.enabled: true
+```
+
+### Step 1: Set up and connect data sources
+
+Open OpenSearch Dashboards and follow these steps:
+
+1. From the **Home** page, select **Add sample data**. For this tutorial, select **Sample web logs**.
```

Comment:
This step should be last in the list since the data source connection has to be established before installing data (reference [the blog post PR](https://github.com/opensearch-project/project-website/pull/2837) for updated steps).